editor/import/resource_importer_scene.cpp

```diff
@@ -233,13 +233,14 @@ static String _fixstr(const String &p_what, const String &p_str) {
 	return what;
 }
 
-static void _pre_gen_shape_list(const Ref<EditorSceneImporterMesh> &mesh, List<Ref<Shape3D>> &r_shape_list, bool p_convex) {
+static void _pre_gen_shape_list(Ref<EditorSceneImporterMesh> &mesh, Vector<Ref<Shape3D>> &r_shape_list, bool p_convex) {
 	ERR_FAIL_NULL_MSG(mesh, "Cannot generate shape list with null mesh value");
 	if (!p_convex) {
 		Ref<Shape3D> shape = mesh->create_trimesh_shape();
 		r_shape_list.push_back(shape);
 	} else {
-		Vector<Ref<Shape3D>> cd = mesh->convex_decompose();
+		Vector<Ref<Shape3D>> cd;
+		cd.push_back(mesh->get_mesh()->create_convex_shape(true, /*Passing false, otherwise VHACD will be used to simplify (Decompose) the Mesh.*/ false));
```

Comment:
Does this change make a difference in the way things are imported by default? It used to convex decompose by default, as a reasonable compromise between speed and accuracy.

Reply:
It uses the `Simple Convex` approach, that using the Convex Hull, generates the convex shape assuming that the passed mesh is meant to be already an optimized convex.

This assumption is given by the fact that the artist/dev will need to explicitly add to the mesh name: `-convcol`, to trigger this code path, so it's reasonable that such shape is optimized to be a convex collision.

If this behavior is bad, and we want to use VHACD also here, I can change it, no problem.

Reply:
It's not that easy to decide, because from what I understand there are actually two scenarios for `-convcol`/`convcolonly`:
1. Some users make a mesh without caring too much if it's convex or concave, and expect the import process to make it usable for physics.
2. Some other users make it convex on purpose, and expect the import process to keep it intact for accuracy purpose.

I would tend to support the case 1 by default, because that's the case users with less knowledge about physics will hit. Users in case 2 know what they are doing so they mostly just need some options to support this case.

So I would rather keep convex decomposition by default, and my plan is to add at least another variant like `-singleconvcol`/`-singleconvcolonly` to handle the other use case.

Does that make sense to you?